crates/holochain_state/src/source_chain.rs

```diff
@@ -445,9 +448,20 @@ impl SourceChain {
             for header in headers {
                 insert_header(txn, header)?;
             }
-            for (op, op_hash, op_order, timestamp) in ops {
+            for (op, op_hash, op_order, timestamp, visibility) in ops {
+                let op_type = op.get_type();
                 insert_op_lite(txn, op, op_hash.clone(), true, op_order, timestamp)?;
-                set_validation_status(txn, op_hash, holochain_zome_types::ValidationStatus::Valid)?;
+                set_validation_status(
+                    txn,
+                    op_hash.clone(),
+                    holochain_zome_types::ValidationStatus::Valid,
+                )?;
+                // TODO: SHARDING: Check if we are the authority here.
+                if !(op_type == DhtOpType::StoreEntry
+                    && visibility == Some(EntryVisibility::Private))
```

Comment:
Maybe add a comment to this condition? I don't know why private entries don't get to be awaiting integration

Reply:
Added a comment but yeh basically we don't gossip or publish private store entry ops